### ISSUE TITLE
CriticMarkup for Sublime Text's Package Control

### ISSUE THREAD

**Issue by elmimmo** [CriticMarkup/CriticMarkup-toolkit#26]
I'd be nice to be able to install CriticMarkup in Sublime Text using [Package Control](https://sublime.wbond.net)

**Comment by mberasategi** [comment]
+1 for this one as well.

**Comment by nathany** [comment]
First thing I did upon discovering CriticMarkup and a Sublime Text plugin. 

Command-Shift-P -> Install -> Crit
oh :frowning:

**Comment by reagle** [comment]
Is this project still alive? Lots of old but simple issues are still open.

**Comment by philipbelesky** [comment]
Agree, I'm trying to test theme support and would love to have this available on package control. Happy to handle submitting the package if you're ok with me doing so via a fork.

**Comment by reagle** [comment]
@philipbelesky, I'd go for it, there's nothing happening here.

**Comment by RoyiAvital** [comment]
Any update on this?

**Comment by beam-LIGHT** [comment]
Submit package
